docs/advanced-topics/wes/wes-wdl-agc-tutorial.rst

```diff
@@ -38,8 +38,9 @@ The following WDL workflow tutorials will cover:
                 "workflowInputs": <pathToInputJSON>
             }
 
-    2. AWS AGC references descriptor files by URLs. The Dockstore CLI references the primary descriptor in an execution request as a GA4GH Tool Registry Service (`TRS <https://github.com/ga4gh/tool-registry-service-schemas>`_) URL.
-    The AGC infrastructure will only be able to access the file referenced by the TRS URL if the workflow is published (i.e. public) on Dockstore.
+    2. AWS AGC references descriptor files by URLs. The Dockstore CLI references the primary descriptor in an execution request as a GA4GH Tool Registry Service (`TRS <https://github.com/ga4gh/tool-registry-service-schemas>`_) URL.  The AGC infrastructure will only be able to access the file referenced by the TRS URL if the workflow is published (i.e. public) on Dockstore.
+
+    3. On AWS AGC, the test parameter file for a WDL workflow must reference other files using S3 URLs.  Other types of URLs, including `https:`, are not supported.
```

Comment:
The linked issue only seems to support cromwell and the `agc-project.yaml`. Maybe this note should limit itself to cromwell just in case miniwdl is supported someday and different  like Toil is